### ISSUE TITLE
brew receipe for terragrunt@0.18

### ISSUE THREAD

**Issue by ebarault** [gruntwork-io/terragrunt#886]
Hi @brikis98,

Terraform publishes 2 different receipes of terraform to homebrew: terraform and terraform@0.11.

What do you think of publishing a terragrunt@0.18 version to homebrew in order to ease working at the same time with both terraform/terragrunt combinations?

**Comment by yorinasub17** [comment]
Homebrew does not support packaging versions that are no longer actively maintained, and we don't really have an intention to support it going forward. See the discussion in https://github.com/Homebrew/homebrew-core/pull/43337.

With that said, you can use the approach described here (https://blog.gruntwork.io/installing-multiple-versions-of-terraform-with-homebrew-899f6d124ff9) for managing multiple versions of terragrunt, or [tgenv](https://github.com/cunymatthieu/tgenv).

**Comment by ebarault** [comment]
I see...
Well of course I came up with my solution to have both latest terragrunt/terraform releases as well as fixed terragrunt@0.18/terraform@0.11 legacy releases.
I just find a bit of a shame that having a terraform@0.11 recipe in Homebrew, we don't have a terragrunt@0.18 version to ease the install while big projects are being migrated.

Because after all, what is "active support" of a wrapping tool that will have no additional features and is fixed to a specific release of terraform? 

Someone could as well have answered that there was a will to provide support and the recipe might be in Homebrew now. 

I did contribute a few PRs and discussions here and I fully grasp the meaning of "community help required". We are a lot to use Terragrunt along with terraform and I'd been willing to help for the sake of having a formula.

Anyway, no big deal though. Nothing prevents anyone to install any version of terragrunt without the need for Homebrew 😄.

Thanks for taking the time to respond, we'll continue our path with tiny wrappers over tiny wrappers.
